mono_repo/lib/src/commands/github/github_yaml.dart

```diff
@@ -345,18 +351,18 @@ extension on CIJobEntry {
 ///
 /// [additionalCacheKeys] is used to create a unique key used to store and
 /// retrieve the cache.
-Map<String, dynamic> _githubJobYaml(
+Job _githubJobYaml(
```

Comment:
Consider renaming this to `_githubJob` since it is no longer returning a raw Map representing yaml